test/crypto-service/test_accountUpdateTransaction.js

```diff
@@ -0,0 +1,1045 @@
+          commonTransactionParams: {
+            signers: [
+              accountPrivateKey
+            ]
+          }
+        });
+        if (response.status === "NOT_IMPLEMENTED") this.skip();
+      } catch (err) {
+        assert.equal(err.data.status, "AUTORENEW_DURATION_NOT_IN_RANGE");
+        return;
+      }
+
+      // The test failed, no error was thrown.
+      assert.fail("Should throw an error");
+    });
+
+    it("(#5) Updates the auto-renew period of an account to the maximum period of 8,000,001 seconds", async function () {
+      // Attempt to update the auto-renew period of the account to 8,000,001 seconds.
+      const autoRenewPeriodSeconds = 8000001;
+      const response = await JSONRPCRequest("updateAccount", {
+        accountId: accountId,
+        autoRenewPeriod: autoRenewPeriodSeconds,
+        commonTransactionParams: {
+          signers: [
+            accountPrivateKey
+          ]
+        }
+      });
+      if (response.status === "NOT_IMPLEMENTED") this.skip();
+
+      // Verify the account was updated with an auto-renew period set to 8,000,001 seconds.
+      verifyAccountAutoRenewPeriodUpdate(autoRenewPeriodSeconds);
+    });
+
+    it("(#6) Updates the auto-renew period of an account to the maximum period plus one second (8,000,002 seconds)", async function () {
+      try {
+        // Attempt to update auto-renew period of the account to 8,000,002 seconds. The network should respond with an AUTORENEW_DURATION_NOT_IN_RANGE status.
+        const response = await JSONRPCRequest("updateAccount", {
+          accountId: accountId,
+          autoRenewPeriod: 8000002,
+          commonTransactionParams: {
+            signers: [
+              accountPrivateKey
+            ]
+          }
+        });
+        if (response.status === "NOT_IMPLEMENTED") this.skip();
+      } catch (err) {
+        assert.equal(err.data.status, "AUTORENEW_DURATION_NOT_IN_RANGE");
+        return;
+      }
+
+      // The test failed, no error was thrown.
+      assert.fail("Should throw an error");
+    });
+  });
+
+  describe("Expiration Time", async function () {
+    async function verifyAccountExpirationTimeUpdate(expirationTime) {
+      // If the account was updated successfully, the queried account's expiration times should be equal.
+      expect(expirationTime).to.equal(await consensusInfoClient.getAccountInfo(accountId).expirationTime);
+      expect(expirationTime).to.equal(await mirrorNodeClient.getAccountData(accountId).accounts[0].expiry_timestamp);
+    }
+
+    it("(#1) Updates the expiration time of an account to 8,000,001 seconds from the current time", async function () {
+      // Attempt to update the expiration time of the account to 8,000,001 seconds from the current time.
+      const expirationTimeSeconds = (Date.now() / 1000) + 8000001;
+      const response = await JSONRPCRequest("updateAccount", {
+        accountId: accountId,
+        expirationTime: expirationTimeSeconds,
```

Comment:
The `expirationTime` should be `int64` as specified in the spec, but in `#1` and `#3` you are passing float value, eg `1731100809.21`

Reply:
Fixed